content/en/docs/demo/_index.md

```diff
@@ -54,6 +54,7 @@ found here:
 - [Recommendation Service](services/recommendation/)
 - [Shipping Service](services/shipping/)
 - [Image Provider Service](services/imageprovider/)
+- [React Native App](services/reactnativeapp/)
```

Comment:
We also need to add the Reactive Native App to the table above.

Reply:
added, I put it in a new row for TypeScript: https://github.com/open-telemetry/opentelemetry.io/pull/5704/commits/8304ec42659ada7c7d1cb77dfa43b1abac0a8c0d